plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/MachineProviderImpl.java

```diff
@@ -294,6 +297,10 @@ public Instance startService(String namespace,
 
             docker.startContainer(StartContainerParams.create(container));
 
+            if (!Strings.isNullOrEmpty(getUser())) {
+                changeWorkingDirOwner(container, machineLogger);
```

Comment:
Is it OK that previous user in container is already running main process of container?